great_expectations/render/renderer_configuration.py

```diff
@@ -75,9 +74,6 @@ class Config:
         validate_assignment = True
         arbitrary_types_allowed = True
 
-    def __len__(self) -> int:
-        return len(self.__fields__)
```

Comment:
Typically there isn't actually any problem with accessing class methods/attributes from an instance method. I suspect this is because of some dynamic programing pydantic is doing.

But based on your example you should be able to rewrite `__len__` like this...

```python
    def __len__(self) -> int:
        return len(self.__clas__.__fields__)
```